tests/loader_wsi_tests.cpp

```diff
@@ -1112,9 +1112,9 @@ TEST(WsiTests, MultiPlatformGetPhysicalDeviceSurfaceSupportKHR) {
         inst.CheckCreate();
 
         auto phys_devs = inst.GetPhysDevs();
-        // Physical devices are enumerated in reverse order to the ICD order
-        VkPhysicalDevice xcb_physical_device = phys_devs[1];
-        VkPhysicalDevice wayland_physical_device = phys_devs[0];
+        // Physical devices are enumerated in order to the ICD order
```

Comment:
This language became a bit mangled. I think you can simplify to "Physical devices are enumerated in ICD discovery order" or similar.